samples/synchronization/sample.yaml

```diff
@@ -12,3 +12,14 @@ tests:
       regex:
         - "threadA: Hello World from (.*)!"
         - "threadB: Hello World from (.*)!"
+  sample.synchronization.openocd:
+    build_on_all: true
```

Comment:
remove the build_on_all, this will really build on every board we have in zephyr, by default

Reply:
Done